argv.js

```diff
@@ -1,9 +1,23 @@
 const fs = require("fs");
 const path = require("path");
 const commandLineArgs = require("command-line-args");
+const pluralize = require("pluralize");
 
 const flags = JSON.parse(fs.readFileSync(path.join(__dirname, "flags.json"), "utf8"));
 
+function isFlag(argument) {
+	// e.g., --test and -t should match
+	return /^(--|-)\w*$/.test(argument);
+}
+
+function throwIfUnknownFlag(unknownArguments) {
+	const flags = unknownArguments ? unknownArguments.filter(isFlag) : [];
```

Comment:
The conditional can be replaced with default arguments in ES6: `function throwIfUnknownFlag(unknownArguments = []) { ...`

Reply:
How I did not see this is beyond me, thanks